integration/spark2/src/test/scala/org/apache/carbondata/datamap/bloom/BloomCoarseGrainDataMapSuite.scala

```diff
@@ -37,6 +39,11 @@ class BloomCoarseGrainDataMapSuite extends QueryTest with BeforeAndAfterAll with
   val dataMapName = "bloom_dm"
 
   override protected def beforeAll(): Unit = {
+    val path: String = new File(
+      classOf[DiskBasedDMSchemaStorageProvider].getResource("/").getPath + "../").getCanonicalPath
+      .replaceAll("\\\\", "/")
```

Comment:
Also check if there is any generic location that we can give here...may be you can take reference from the spark UT framework where we add the location in spark-common